docs/getting-started/csm-stack/csm-sandbox.ipynb

```diff
@@ -0,0 +1,257 @@
+    "def plot_footprint(lons, lats, aspect):\n",
+    "    plt.close()\n",
+    "    plt.rcParams[\"figure.figsize\"] = aspect\n",
+    "    plt.axes().set_aspect('equal','datalim')\n",
+    "    plt.plot(lons, lats)\n",
+    "    plt.xlabel(\"Longitude (deg)\")\n",
+    "    plt.ylabel(\"Latitude (deg)\")\n",
+    "    plt.title(\"CSM footprint\")\n",
+    "    plt.show()\n",
+    "\n",
+    "def plot_footprint_comparison(fp1, fp2, aspect):\n",
+    "    plt.clf()                                   # clear previous figure\n",
+    "    plt.rcParams[\"figure.figsize\"] = aspect     # set aspect ratio of plot\n",
+    "    plt.axes().set_aspect('equal','datalim')    \n",
+    "    fp1_plot, = plt.plot(fp1[0], fp1[1], 'b')   # Plot footprint 1 in blue\n",
+    "\n",
+    "    margin = 0.001                              # boundaries centered on footprint 1\n",
+    "    fp1_lim_x = plt.xlim()\n",
+    "    fp1_lim_y = plt.ylim()\n",
+    "    fp1_lim_x = (fp1_lim_x[0] - margin, fp1_lim_x[1] + margin)\n",
+    "    fp1_lim_y = (fp1_lim_y[0] - margin, fp1_lim_y[1] + margin)\n",
+    "\n",
+    "    fp2_plot, = plt.plot(fp2[0], fp2[1], 'r')       # Plot footprint 2 in red\n",
+    "    plt.title(\"Original vs. Modified Footprint\")    # Title and axis labels\n",
+    "    plt.xlabel(\"Longitude (deg)\")\n",
+    "    plt.ylabel(\"Latitude (deg)\")\n",
+    "    fp1_plot.set_label(fp1[2])  # Labels/Legend\n",
+    "    fp2_plot.set_label(fp2[2])\n",
+    "    plt.legend()\n",
+    "    plt.xlim(fp1_lim_x)         # Set Size\n",
+    "    plt.ylim(fp1_lim_y)\n",
+    "    plt.show()                  # Show plot"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Stats/Footprint of original ISD"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "focal_length_model:      {'focal_length': 352.9271664}\n",
+      "detector_center:         {'line': 0.430442527, 'sample': 2542.96099}\n",
+      "optical_distortion:      {'radial': {'coefficients': [-0.0073433925920054505, 2.8375878636241697e-05, 1.2841989124027099e-08]}}\n",
+      "center_ephemeris_time:   297088762.2425226\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Load Dict from JSON-style ISD File\n",
+    "data_dir = '../data/image_to_ground'\n",
+    "isd_file = os.path.join(data_dir, 'isd_file.json')\n",
+    "\n",
+    "with open(isd_file) as json_file:\n",
+    "    isd_dict = json.load(json_file)\n",
+    "\n",
+    "# Print selected values from ISD\n",
+    "print_stats(isd_dict, ('focal_length_model', 'detector_center', 'optical_distortion', 'center_ephemeris_time'))\n",
+    "\n",
+    "# Create Camera Model\n",
+    "camera = csm.create_csm(isd_file)\n",
+    "\n",
+    "# Get the footprint using the model\n",
+    "boundary = csm.generate_boundary((isd_dict[\"image_lines\"], isd_dict[\"image_samples\"]))\n",
```

Comment:
I noticed this ISD was 1x1 pixel. It would be better to do something that is full sample length with cropped lines. Especially since this is a CTX camera.